hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/batch/DirectBufferPool.java

```diff
@@ -14,7 +16,6 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package pl.allegro.tech.hermes.consumers.consumer.batch;
```

Comment:
Can we exclude this file from the validation? The file has been copied from an external library, so it makes sens to keep the original formatting.